js/src/cli/docker-compose.yaml

```diff
@@ -2,20 +2,26 @@ version: "3"
 services:
   langchain-playground:
     image: langchain/${_LANGSMITH_IMAGE_PREFIX-}langchainplus-playground:latest
+    ports:
+      - 3001:3001
   langchain-frontend:
-    image: langchain/${_LANGSMITH_IMAGE_PREFIX-}langchainplus-frontend:latest
+    image: langchain/${_LANGSMITH_IMAGE_PREFIX-}langchainplus-frontend-dynamic:latest
```

Comment:
is frontend-dynamic the preferred frontend image now? People will likely not want to change the backend url when running with compose